build.sbt

```diff
@@ -31,14 +29,16 @@ lazy val microservice = (project in file("."))
   .disablePlugins(sbt.plugins.JUnitXmlReportPlugin)
   .configs(testConfig: _*)
   .settings(
+    scalaVersion := "2.13.10",
+    targetJvm := "jvm-11",
     commonSettings,
     unitTestSettings,
     integrationComponentTestSettings,
     allTest,
     scoverageSettings
   )
   .settings(majorVersion := 0)
-  .settings(scalacOptions += "-P:silencer:pathFilters=routes")
+  .settings(scalacOptions += "-Wconf:cat=unused-imports&src=target/scala-2\\.13/routes/.*:s")
```

Comment:
This is to stop these warnings right?
[warn] /home/ian/work/hmrc/customs-notifications-receiver-stub/conf/routes: Unused import

Reply:
think so yeah